lib/utils.js

```diff
@@ -20,7 +20,15 @@ const labelParser = function (labels) {
 }
 /* Fingerprinting */
 const shortHash = require('short-hash')
-const fingerPrint = function (text, hex) {
+const xxh = require('xxhashjs')
+const xxhSeed = 'cloki'
+const fingerPrint = function (text, hex, alg) {
+  alg = alg || process.env.HASH || 'short-hash'
+  switch (alg) {
+    case 'XXH':
+      if (hex) return xxh.h64(text, xxhSeed).toString(16)
```

Comment:
To keep compatibility with clickhouse seed is not needed

```suggestion
const fingerPrint = function (text, hex, alg) {
  alg = alg || process.env.HASH || 'short-hash'
  switch (alg) {
    case 'XXH':
      if (hex) return xxh.h64(text, '').toString(16)
```